DistributedTransaction/DistributedTransaction.tla

```diff
@@ -381,36 +387,63 @@ ServerCommit ==
 ServerCleanupStaleLock ==
   \E k \in KEY :
     \E l \in key_lock[k] :
-      /\ SendReqs({[type |-> "cleanup",
+      /\ SendReqs({[type |-> "check_txn_status",
                     start_ts |-> l.ts,
-                    primary |-> l.primary]})
+                    primary |-> l.primary,
+                    resolving_pessimistic_lock |-> l.type = "lock_key"
+                    ]})
       /\ UNCHANGED <<resp_msgs, client_vars, key_vars, next_ts>>
 
-\* Clean up stale locks by checking the status of the primary key.  Commmit
-\* the secondary keys if primary key is committed; otherwise rollback the
-\* transaction by rolling-back the primary key, and then also rollback the
-\* secondarys.
-ServerCleanup ==
+\* Clean up stale locks by checking the status of the primary key.  It
+\* can be divided into two cases: pk_lock exists or not.  Note that it is 
+\* hard to model the TTL in TLA+ spec, so instead, the TTL is considered 
+\* constantly expired when the action is taken.
+\* The client does not care about the resp message, it cares about whether
+\* the lock is resolved, and resolving the lock is actually resolve the 
+\* status of the corresponding transaction.  Since, in the TLA+ spec,
+\* we ignored the `check_txn_status` response message.

```

Comment:
```suggestion
\* Clean up stale transaction on checking the status of the primary key.
\*
\* In practice, the transaction will be rolled back if TTL on the lock in expired.  But
\* because it is hard to model the TTL in TLA+ spec, the TTL is considered constantly
\* expired when the action is taken.
\*
\* Moreover, TiKV will send a response for `TxnStatus` to the client, and then depending
\* on the `TxnStatus`, the client will send `resolve_rollback` or `resolve_commit` to the
\* secondary keys to clean up stale locks.  In the TLA+ spec, the reponse to `check_txn_status`
\* is omitted and TiKV will directly send `resolve_rollback` or `resolve_commit` message to
\* secondary keys, because the action of client sending resolve message by proxying the
\* `TxnStatus` from TiKV does not change the state of client, therefore is equal to directly
\* sending resolve message by TiKV
```

Reply:
DistributedTransaction: CheckTxnStatus update comments.